test/expected/issue_50_delete_filters.out

```diff
@@ -0,0 +1,170 @@
+     "type": "DELETE",                             +|                |                                        | 
+     "table": "notes",                             +|                |                                        | 
+     "schema": "public",                           +|                |                                        | 
+     "columns": [                                  +|                |                                        | 
+         {                                         +|                |                                        | 
+             "name": "id",                         +|                |                                        | 
+             "type": "int4"                        +|                |                                        | 
+         },                                        +|                |                                        | 
+         {                                         +|                |                                        | 
+             "name": "body",                       +|                |                                        | 
+             "type": "text"                        +|                |                                        | 
+         }                                         +|                |                                        | 
+     ],                                            +|                |                                        | 
+     "old_record": {                               +|                |                                        | 
+         "id": 2                                   +|                |                                        | 
+     },                                            +|                |                                        | 
+     "commit_timestamp": "2000-01-01T08:01:01.000Z"+|                |                                        | 
+ }                                                  |                |                                        | 
+(2 rows)
+
+----------------------------------------------------------------------------------------
+-- When Replica Identity is Not Full, only filters referencing the pkey are respected --
+----------------------------------------------------------------------------------------
+alter table public.notes replica identity full;
+insert into public.notes(id, body)
+values
+    (1, 'bbb'),
+    (2, 'ccc');
+select clear_wal();
+ clear_wal 
+-----------
+ 
+(1 row)
+
+delete from public.notes;
+select
+    rec,
+    is_rls_enabled,
+    subscription_ids,
+    errors
+from
+   walrus;
+                        rec                         | is_rls_enabled |            subscription_ids            | errors 
+----------------------------------------------------+----------------+----------------------------------------+--------
+ {                                                 +| f              | {f4539ebe-c779-5788-bbc1-2421ffaa8954} | {}
```

Comment:
after setting `replica identity full` the filter on `body = 'bbb'` works, so we get `subscription_id`